dist/index.html

```diff
@@ -7,7 +7,7 @@
   <title>You Travel Tracker</title>
 </head>
 <body>
-  <header>
+  <header class="header">
```

Comment:
Thank you @wlffann for pulling me out of this. It could have been a huge rabbit hole!